x-pack/plugins/apm/public/hooks/use_fetcher.tsx

```diff
@@ -30,6 +30,12 @@ export const isPending = (fetchStatus: FETCH_STATUS) =>
   fetchStatus === FETCH_STATUS.LOADING ||
   fetchStatus === FETCH_STATUS.NOT_INITIATED;
 
+export const isFailure = (fetchStatus: FETCH_STATUS) =>
+  fetchStatus === FETCH_STATUS.FAILURE;
+
+export const isSuccess = (fetchStatus: FETCH_STATUS) =>
+  fetchStatus === FETCH_STATUS.SUCCESS;
```

Comment:
We could probably remove the `export` from the `FETCH_STATUS` definition to force consumers to use these helpers. Not in scope for this PR though.